jobs/dynatrace-oneagent-windows/templates/start.ps1

```diff
@@ -162,6 +163,14 @@ function setHostTags() {
 	Set-Content -Path $hostTagsFile -Value $cfgHostTags
 }
 
+function setHostProps() {
+	$hostPropsFile = "${configDir}\hostcustomproperties.conf"
+
+	# We need to save the file content even if it's empty.
```

Comment:
Why not add an 'if' here to check if cfgHostProps is set and avoid writing an empty file?